packages/angular-cli/models/webpack-build-production.ts

```diff
@@ -36,22 +35,6 @@ export const getWebpackProdConfigPartial = function(projectRoot: string,
           algorithm: 'gzip',
           test: /\.js$|\.html$|\.css$/,
           threshold: 10240
-      }),
-      // LoaderOptionsPlugin needs to be fully duplicated because webpackMerge will replace it.
-      new webpack.LoaderOptionsPlugin({
-        test: /\.(css|scss|sass|less|styl)$/,
```

Comment:
This test technically isn't needed and It adds an additional check per resource request.